databroker/broker.py

```diff
@@ -1068,10 +1074,10 @@ def inner(stream):
 
 def header_io(db_in, db_out):
     def outer(f):
-        def inner(header):
+        def inner(headers):
```

Comment:
We should discuss how to handle multiple input streams if you have time at some point (are they handed in as a tuple, args, dict, or kwargs?)